pkg/storage/etcd/testing/utils.go

```diff
@@ -113,7 +113,13 @@ func configureTestCluster(t *testing.T, name string) *EtcdTestServer {
 		t.Fatal(err)
 	}
 
-	m.CertificatesDir, err = ioutil.TempDir(os.TempDir(), "etcd_certificates")
+	// Allow test launches to control where etcd data goes, for space or performance reasons
+	baseDir := os.Getenv("TEST_ETCD_DIR")
+	if len(baseDir) == 0 {
+		baseDir = os.TempDir()
+	}
+
+	m.CertificatesDir, err = ioutil.TempDir(baseDir, "etcd_certificates")
```

Comment:
`ioutil.TempDir` uses `os.TempDir()` automatically if the base dir is empty string.